etcdserver/etcdserverpb/rpc.proto

```diff
@@ -658,6 +665,25 @@ message LeaseKeepAliveResponse {
   int64 TTL = 3;
 }
 
+message LeaseTimeToLiveRequest {
+  // ID is the lease ID for the lease.
+  int64 ID = 1;
+  // keys is true to query all the keys attached to this lease.
+  bool keys = 2;
+}
+
+message LeaseTimeToLiveResponse {
+  ResponseHeader header = 1;
+  // ID is the lease ID from the keep alive request.
+  int64 ID = 2;
+  // TTL is the remaining TTL in seconds for the lease.
+  double TTL = 3;
```

Comment:
can be change this back to int64? what is the reason to make this double? this change is inconsistent with other ttl fields.